charts/spiderpool/templates/multus-daemonset.yaml

```diff
@@ -212,6 +212,15 @@ spec:
             {{- if .Values.multus.multusCNI.extraVolumes }}
             {{- include "tplvalues.render" ( dict "value" .Values.multus.multusCNI.extraVolumeMounts "context" $ ) | nindent 12 }}
             {{- end }}
+          lifecycle:
```

Comment:
建议加个 multus values 选项，unstaill: false
默认关闭

cilium 类似参考
```
cni:
  # -- Install the CNI configuration and binary files into the filesystem.
  install: true

  # -- Remove the CNI configuration and binary files on agent shutdown. Enable this
  # if you're removing Cilium from the cluster. Disable this to prevent the CNI
  # configuration file from being removed during agent upgrade, which can cause
  # nodes to go unmanageable.
  uninstall: false
```